deployment/conf/.templates/deployment.cfg.templ

```diff
@@ -59,3 +59,5 @@ reboot.mode={{ default .Env.reboot_mode "false" }}
 
 condor.mode={{ default .Env.condor_mode "1" }}
 condor.submit.desc.file.path={{ default .Env.condor_submit_desc_file_path "/kb/deployment/misc/" }}
+condor-submit-workdir={{ default .Env.condor.submit.workdir "/mnt/awe/condor" }}
```

Comment:
Already merged in 121.. Will make branches based off of develop, instead of other branches from now on.

Reply:
For the default in the final, let's make this /mnt/condor.